common/src/main/java/com/sequenceiq/cloudbreak/telemetry/databus/AbstractDatabusStreamConfiguration.java

```diff
@@ -17,6 +19,11 @@ public AbstractDatabusStreamConfiguration(boolean enabled, String dbusAppName, S
         this.dbusStreamName = dbusStreamName;
     }
 
+    @Override
+    public boolean isStreamingEnabled() {
+        return isEnabled();
+    }
+
     public boolean isEnabled() {
         return enabled;
     }
```

Comment:
isStreamingEnabled could be an abstract function or just return false by default in the abstract class....enabled flag is for the feature, streaming config is for the stream processing. e.g.: metering is enabled in datahub but streaming should not enbled there (of course it won't start any processing without using the processor but it will create at least a worker thread which waits for processing data)

Reply:
I understand that, but in that case shouldn't isStreamingEnabled and isEnabled have their separate variables so they can be controlled separately?

Reply:
normally that variable should be on the implementation side, as you need to implement that, probably removing that from here (from the abstract class) or just make it false make the most sense in order to avoid a default not desired behaviour

Reply:
That's fine by me both ways. I just wanted to make sure that the default behaviour don't depend on each other causing some undesired outcomes in child classes not overriding the function. So an override should be added to the child classes and either a default false or nothing here in the abstract.